src/main/java/core/basesyntax/AreaObtanable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface AreaObtanable {
```

Comment:
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-verbs-for-classinterface-names
```suggestion
interface AreaCalculator {
```